internal/databases/postgres/pagefile_test.go

```diff
@@ -57,6 +54,47 @@ var allBlocksTestIncrement = newTestIncrement(smallLSN)
 // so created increment consists of zero blocks
 var zeroBlocksTestIncrement = newTestIncrement(bigLSN)
 
+func TestIsPagedFile_Directory(t *testing.T) {
```

Comment:
It is better to use parametrized test cases like here https://github.com/wal-g/wal-g/blob/01fbb7132636bf0da71eec5caa642d790f3536c5/internal/databases/postgres/query_runner_test.go#L68